From 6abdbf1aa571e28c8379cd0a2b3e12f2ee7a2cda Mon Sep 17 00:00:00 2001
From: Tomasz Figa <t.figa@samsung.com>
Date: Tue, 21 May 2013 18:28:10 +0200
Subject: [PATCH 0240/1302] ARM: dts: exynos4210-trats: Add aliases for
 bit-banged I2C buses

This patch adds missing aliases of bit-banged I2C buses required for
numbered I2C bus registration.

Signed-off-by: Tomasz Figa <t.figa@samsung.com>
Signed-off-by: MyungJoo Ham <myungjoo.ham@samsung.com>
---
 arch/arm/boot/dts/exynos4210-trats.dts | 9 +++++++--
 1 file changed, 7 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/exynos4210-trats.dts b/arch/arm/boot/dts/exynos4210-trats.dts
index 4ebf3e6..7dc92cf 100644
--- a/arch/arm/boot/dts/exynos4210-trats.dts
+++ b/arch/arm/boot/dts/exynos4210-trats.dts
@@ -19,6 +19,11 @@
 	model = "Samsung Trats based on Exynos4210";
 	compatible = "samsung,trats", "samsung,exynos4210";
 
+	aliases {
+		i2c8 = &i2c_s5k5baf;
+		i2c9 = &i2c_max17042;
+	};
+
 	memory {
 		reg =  <0x40000000 0x10000000
 			0x50000000 0x10000000
@@ -187,7 +192,7 @@
 		};
 	};
 
-	i2c-gpio {
+	i2c_s5k5baf: i2c-gpio {
 		compatible = "i2c-gpio";
 		gpios = <&gpc1 0 0>, <&gpc1 2 0>;
 		i2c-gpio,delay-us = <2>;
@@ -500,7 +505,7 @@
 		};
 	};
 
-	i2c-gpio-0 {
+	i2c_max17042: i2c-gpio-0 {
 		compatible = "i2c-gpio";
 		gpios = <&gpy4 0 0>, <&gpy4 1 0>;
 		i2c-gpio,delay-us = <5>;
-- 
1.8.3.2

